k9mail/src/main/java/com/fsck/k9/planck/infrastructure/threading/EngineThreadLocal.kt

```diff
@@ -19,10 +19,7 @@ class EngineThreadLocal private constructor(
     }
 
     private fun configureOnEveryCall(engine: Engine) {
-        engine.config_passphrase_for_new_keys(
```

Comment:
why is this function call not needed anymore? Is it because now you ask on app startup?

Reply:
This was the old, unwanted behavior. We were setting on every new core instance the passphrase for new keys we had saved as a setting in the app. Now we don't want this setting, and we don't want to persist it. So everything is gone.

Reply:
ok sounds good